frontend/src/main/web/lib/components/glossary/DeleteEntry.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import {PureRenderMixin} from 'react/addons';
+import Actions from '../../actions/GlossaryActions';
+import { Button, Icon, Tooltip, Overlay } from 'zanata-ui';
+
+var DeleteEntry = React.createClass({
```

Comment:
What is a "delete entry"? The name is not enough to understand so this could use a doc comment to briefly explain.